src/Files.App/Helpers/Application/AppLifecycleHelper.cs

```diff
@@ -322,5 +322,20 @@ public static void HandleAppUnhandledException(Exception? ex, bool showToastNoti
 			}
 			Process.GetCurrentProcess().Kill();
 		}
+
+		/// <summary>
+		///	Checks if the taskbar is set to auto-hide.
+		/// </summary>
+		public static bool IsAutoHideTaskbarEnabled()
+		{
+			const string registryKey = @"Software\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3";
+			const string valueName = "Settings";
+
+			using (var key = Microsoft.Win32.Registry.CurrentUser.OpenSubKey(registryKey))
+			{
+				var value = key?.GetValue(valueName) as byte[];
+				return value != null && ((value[8] & 0x01) == 1); // The least significant bit of the 9th byte controls the auto-hide setting
+			}
```

Comment:
Decouple using statement and put comment in a new line.

```suggestion
			using var key = Microsoft.Win32.Registry.CurrentUser.OpenSubKey(registryKey);

			var value = key?.GetValue(valueName) as byte[];

			// The least significant bit of the 9th byte controls the auto-hide setting																		
			return value != null && ((value[8] & 0x01) == 1);
```